lib/interface.rb

```diff
@@ -0,0 +1,81 @@
+require_relative 'menu'
```

Comment:
great taht you have separated the the ui into a separate class - but the client didn't ask for any of this in the brief.